extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -96,7 +96,7 @@ class CornerstoneViewportService implements IViewportService {
   }
 
   public getViewportId(viewportIndex: number): string {
-    return `viewport-${viewportIndex}`;
+    return this.viewportsInfo[viewportIndex]?.getViewporId?.();
```

Comment:
This has a typo

Reply:
Where?  The ?. is intentional to avoid calling an undefined value.